docs/modules/project/pages/continuous-integration.adoc

```diff
@@ -3,51 +3,68 @@
 
-== Publish the artifacts
+== Release and publish the artifacts
 
-Artifacts are published to Maven Central and jCenter by way of Bintray's _Distribution as a Service_ platform.
-
-Before publishing, you need to configure your gpg signing and Bintray credentials.
-Create the file [path]_$HOME/.gradle/gradle.properties_ and populate the following properties.
+To build and publish a release the following properties have to be set, for example in the file ~/.gradle/gradle.properties:
 
+.~/.gradle/gradle.properties
+[source,properties]
 ----
-signing.keyId=
-signing.password=
+signing.keyId=...              # <1>
+signing.password=...
 signing.secretKeyRingFile=/home/YOUR_USERNAME/.gnupg/secring.gpg
-bintrayUsername=
-bintrayApiKey=
-----
-
-To build, assemble and sign the archives (jars and distribution zip), run:
-
- $ ./gradlew -PpublishRelease=true signJars
-
-TIP: The `publishRelease=true` property is technically only required if the version is a snapshot.
-
-To build, assemble (but not sign) and install the archives (jars and distribution zip) into the local Maven repository, run:
-
- $ ./gradlew -PpublishRelease=true install
-
-To build, assemble, sign and publish the archives (jars and distribution zip) to Bintray, run:
 
- $ ./gradlew clean
-   ./gradlew -i -x pMNPTML asciidoctorj-api:bintrayUpload asciidoctorj:bintrayUpload asciidoctorj-distribution:bintrayUpload
+sdkman_consumer_key=...        # <2>
+sdkman_consumer_token=...
 
-CAUTION: Don't run the `clean` task in the same execution as the `bintrayUpload` because it will not upload one of the signatures.
-
-If you want to first perform a dry run of the upload, add the `dryRun=true` property.
-
- $ ./gradlew -i -PdryRun=true -x pMNPTML asciidoctorj-api:bintrayUpload asciidoctorj:bintrayUpload asciidoctorj-distribution:bintrayUpload
-
-NOTE: The `-x pMNPTML` is necessary to work around a bug in the publishing plugin that prevents it from signing the archives.
-
-IMPORTANT: Bintray does not allow you to publish snapshots.
-You have to first update the version in [path]_gradle.properties_ to a release (or pre-release) version number.
-Currently, Gradle is not configured to automatically tag a release, so *you have to create the git tag manually*.
-
-== Publish snapshot versions
-
-To publish a snapshot version simply run:
+sonatypeUsername=...           # <3>
+sonatypePassword=...
+----
+<1> Settings for signing the artifacts
+<2> Credentials for publishing a new release to sdkman
+<3> Credentials for publishing artifacts to oss.sonatype.org
 
- $ ./gradlew asciidoctorj-api:artifactoryPublish asciidoctorj:artifactoryPublish asciidoctorj-distribution:artifactoryPublish
+The following steps are necessary to build a new release:
 
+1. Update the version in gradle.properties to a release version, i.e. from `2.4.4-SNAPSHOT` to `2.4.4`.
```

Comment:
Should we rely on auto-numering? :roll_eyes:

Reply:
Do you mean to take the concrete version from some computed variable?

Reply:
Ah, now I get what you mean!
I can change it to auto-numbering, sure. 
I personally find the source more readable if the numbers are in, but for consistency it might be better to use auto-numbering if we also apply it elsewhere.